deploy/eks/kiam-server-deplyoyment.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
```

Comment:
nit: change the name of this file to `kiam-server-deployment.yaml`

Reply:
there is a typo in the file name: `deploy/eks/kiam-server-deplyoyment.yaml`